packages/bot-web-ui/src/components/transactions/transaction.tsx

```diff
@@ -10,7 +10,28 @@ import { localize } from '@deriv/translations';
 import { popover_zindex } from 'Constants/z-indexes';
 import { useDBotStore } from 'Stores/useDBotStore';
 
-const TransactionIconWithText = ({ icon, title, message, className }) => (
+type TTransactionIconWithText = {
+    icon: React.ReactElement;
+    title: string;
+    message?: React.ReactNode;
+    className?: string;
+};
+
+type TPopoverItem = {
+    icon?: React.ReactElement;
+    title: string;
+    children: React.ReactNode;
+};
+
+type TPopoverContent = {
+    contract: TContract;
+};
+
+type TTransaction = {
+    contract?: TContract;
+};
+
+const TransactionIconWithText = ({ icon, title, message, className }: TTransactionIconWithText) => (
```

Comment:
I think we can provide here expected type easily
<img width="925" alt="Screenshot 2024-02-21 at 12 23 35" src="https://github.com/binary-com/deriv-app/assets/103181650/07e6d91e-5ffc-404b-aaf7-8064739a1f2b">